Proto/cg.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
@matthid @KevinRansom  Yes, I meant replace with an F# script. F# is a fantastic scripting language and should always have been the default scripting language for all build/test automation in the Visual F# Tools over the years (apart from .fsproj project files and some MSBuild automation).